src/dss_cell.hpp

```diff
@@ -0,0 +1,82 @@
+#pragma once
+#pragma once
+
+#include <vector>
+
+#include <common_types.hpp>
+#include <util/debug.hpp>
+
+namespace nest {
+namespace mc {
+
+/// data spike source: A cell that generates spikes provided as a vector of
+/// floating point valued spiketimes at the start of a run
+
+class dss_cell {
+public:
+    struct dss_cell_description {
```

Comment:
Indeed. But they are always used together and always change together.
I agreed that you want to have them separate to have the recipe produce description and not objects. What is the added benefit of having it in a separate sourcefile?